src/data-access/apiPaths.ts

```diff
@@ -0,0 +1,20 @@
+
+
+export const apiKey: string = `6ef10486c5df46ca61884c8b042d53bd`;
+export const baseUrl: string = `https://api.themoviedb.org/3/`;
+
+//spesifik urller burada
+export const genreListUrl: string = `${baseUrl}genre/movie/list?api_key=${apiKey}`;
```

Comment:
Why do we need to append apiKey in every url? Axios can do it from one place. 

In addition why do we need to add base url all the time? Again, we are already configuring axios to use baseurl.